src/components/header/Header.component.tsx

```diff
@@ -1,2 +1,56 @@
+import cartWhite from '@/assets/img/headerImg/cartWhite.png';
+import login from '@/assets/img/headerImg/login.png';
+import logoMA from '@/assets/img/headerImg/logoMA.png';
+import moon from '@/assets/img/headerImg/moon.png';
+import signUp from '@/assets/img/headerImg/signUp.png';
+import sun from '@/assets/img/headerImg/sun.png';
+
 import styles from './header.module.css';
-export const HeaderComponent = () => <header className={styles.header}>Header</header>;
+export const HeaderComponent = () => (
+    <header className={styles.header}>
+        <img className={styles.logoMA} src={logoMA} alt="Logo Masters academy" />
+        <div className={styles.switcherTheme}>
+            <button className={styles.themeDayBtn}>
+                <img className={styles.themeDayImg} src={sun} alt="Switcher day theme" />
```

Comment:
<img width="993" alt="Screenshot 2024-04-28 at 21 05 52" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/e7723d68-219b-4be1-a108-bb7b7b67f1ad">

I am not sure that icon size from disign? please be more attentive to details